portal/src/main/webapp/config_service.jsp

```diff
@@ -60,6 +60,7 @@
             "show.civic",
             "show.genomenexus",
             "show.mutation_mappert_tool.grch38",
```

Comment:
@leexgh just noticed typo here: `show.mutation_mappert_tool.grch38`. Should be `show.mutation_mapper_tool.grch38`. Can be addressed in separate PR (good to update docs as well)